modules/proc_configuring-notificationsconfiguration-crd-by-using-the-ocp-cli.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-notificationsconfiguration-cr-by-using-cli_{context}"]
+= Configuring the NotificationsConfiguration CR by using CLI
+
+You can configure the `NotificationsConfiguration` CR by using CLI.
```

Comment:
Same comment as above about spelling out CR the first instance in a new section:
```suggestion
You can configure the `NotificationsConfiguration` custom resource (CR) by using CLI.
```

Reply:
Fixed.